.github/workflows/verify.yml

```diff
@@ -55,4 +55,4 @@ jobs:
         uses: golangci/golangci-lint-action@3cfe3a4abbb849e10058ce4af15d205b6da42804 # v4.0.0
         timeout-minutes: 5
         with:
-          version: v1.54
+          version: v1.54.2
```

Comment:
Use v1.57 is the latest minor dont set the patch then the job will always get the latest for the minor